src/main/webapp/app/course/manage/course-management-tab-bar/course-management-tab-bar.component.ts

```diff
@@ -28,7 +29,7 @@ import { CourseAdminService } from 'app/course/manage/course-admin.service';
 @Component({
     selector: 'jhi-course-management-tab-bar',
     templateUrl: './course-management-tab-bar.component.html',
-    styleUrls: ['./course-management-tab-bar.component.scss'],
+    styleUrls: ['../../../shared/tab-bar/tab-bar.scss'],
```

Comment:
I am not sure if this is necessarily better, but I have created a new scss file ./course-management-tab-bar.component.scss that imports the style. I've seen that importing the styles is done in other places, too.

It's high up because it is used in app/course-overview and app/course/manage/course-management-tab-bar. Both components need the tab bar and shared seems like a reasonable place

Reply:
I like that!